docs/source/index.rst

```diff
@@ -6,17 +6,25 @@
 Substra documentation
 =====================
 
+Substra is an open source federated learning (FL) software. It provides a flexible Python library and a web application to run federated learning training at scale.
+
+Substra's main usage is in production environments. It has already been deployed and used by hospitals and biotech companies: see the `MELLODDY <https://www.melloddy.eu/>`_ and the `HealthChain <https://www.nature.com/articles/s41591-022-02155-w>`_ projects.
+
+
 .. image:: banner.jpg
-  :width: 800
+  :width: 600
   :align: center
   :alt: Substra
 
 
+The key Substra differentiators are:
 
-
-Substra is an open source federated learning (FL) software. It provides a flexible Python interface and a web app to run federated learning training at scale.
-
-Substra's main usage is in production environments. It has already been deployed and used by hospitals and biotech companies (see the `MELLODDY <https://www.melloddy.eu/>`_ project for instance). Substra can also be used on a single machine on a virtually split dataset to perform FL simulations and debug code before launching experiments on a real network.
+* **Framework agnostic** Any Python library can be used: PyTorch, TensorFlow, sklearn, etc.
+* **Flexible** Any kind of computation can be run: machine learning, analytics, etc.
+* **Scalable** Support for vertical scaling (several trainings on one machine) and horizontal scaling (training on several machines).
+* **Traceable** All machine learning actions are traced.
```

Comment:
I think it might be bad to use "trace" in both the word and its description, maybe we can say something like this (just a suggestion):

```suggestion
* **Traceable** All machine learning operations are logged in an auditable read-only store.```

Reply:
Thanks for the proposition.

I think it's better. But I find the word `store` weird? Do you have a more understandable / general purpose word? Database maybe?

Reply:
🤔 I suppose a database is a specific implementation of a store but I think database is appropriate.